packages/edit-site/src/components/sidebar/global-styles-v2/components/inspector.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { motion } from 'framer-motion';
+import ReactJson from 'react-json-view';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	FormGroup,
+	Panel,
+	PanelBody,
+	PanelHeader,
+	Select,
+	useNavigatorHistory,
+	useNavigatorLocation,
+	View,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAppState } from '../state';
+
+const routes = [
+	'/',
+	'/colors',
+	'/colors/palette',
+	'/colors/elements/background',
+	'/typography',
+	'/typography/elements/headings',
+].map( ( o ) => ( {
+	value: o,
+	label: o,
+} ) );
+
+export const Inspector = () => {
+	const navigator = useNavigatorHistory();
+	const location = useNavigatorLocation();
+	const currentPath = location?.pathname;
+	const handleOnRouteChange = ( next ) => navigator.push( next );
+	const appState = useAppState();
+
+	return (
+		<div style={ { position: 'fixed', top: 16, left: 16 } }>
+			<motion.div drag dragMomentum={ false }>
+				<Card css={ { minWidth: 400 } } size="small">
+					<CardHeader size="small">Inspector</CardHeader>
+					<CardBody>
+						<FormGroup horizontal label="Go to">
+							<Select
+								onChange={ handleOnRouteChange }
+								options={ routes }
+								value={ currentPath }
+							/>
+						</FormGroup>
+					</CardBody>
+					<Panel>
+						<PanelHeader>State</PanelHeader>
+						<PanelBody>
+							<View
+								css={ {
+									fontFamily: 'monospace',
+									maxWidth: '100%',
+									maxHeight: 300,
+									overflowY: 'auto',
+									fontSize: 12,
+									lineHeight: 1,
+								} }
+							>
+								<ReactJson
```

Comment:
Necessary only for the proof of concept?

Reply:
It's conceivable we might have an "advanced view" where more of theme.json is exposed than what the regular UI reflects, but it's not something to prioritize now.